### ISSUE TITLE
rabbitmq_policy broken in 2018.3.0

### ISSUE THREAD

**Issue by liquidgecka** [saltstack/salt#46880]
### Description of Issue/Question

When rabbitmq_policy attempts to make a change I get an exception the priority is defined as an argument twice.

```
----------
          ID: rabbit_ha_policy
    Function: rabbitmq_policy.present
        Name: ha-all-queues
      Result: False
     Comment: An exception occurred in this state: Traceback (most recent call last):
                File "/usr/lib/python2.7/dist-packages/salt/state.py", line 1878, in call
                  **cdata['kwargs'])
                File "/usr/lib/python2.7/dist-packages/salt/loader.py", line 1823, in wrapper
                  return f(*args, **kwargs)
                File "/usr/lib/python2.7/dist-packages/salt/states/rabbitmq_policy.py", line 108, in present
                  runas=runas)
              TypeError: set_policy() got multiple values for keyword argument 'priority'
     Started: 21:37:25.570994
    Duration: 575.121 ms
     Changes:
```

### Setup

```
rabbit_ha_policy:
  rabbitmq_policy.present:
    - name: ha-all-queues
    - pattern: '.*'
    - definition: '{"ha-mode":"all"}'
    - priority: 0
```

Removing the priority argument from this line (and the one just like it a few lines later) in rabbitmq_priority.py makes this work:
```
            result = __salt__['rabbitmq.set_policy'](vhost,
                                                     name,
                                                     pattern,
                                                     definition,
                                                     apply_to,
                                                     priority=priority,
                                                     runas=runas)
```

### Steps to Reproduce Issue

Setup above, highstate.

### Versions Report

```
Salt Version:
           Salt: 2018.3.0

Dependency Versions:
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: 2.4.2
      docker-py: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.8
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: 1.0.3
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 2.7.12 (default, Dec  4 2017, 14:50:18)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 15.2.0
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.1.4

System Versions:
           dist: Ubuntu 16.04 xenial
         locale: UTF-8
        machine: x86_64
        release: 4.4.0-72-generic
         system: Linux
        version: Ubuntu 16.04 xenial
```

**Comment by garethgreenaway** [comment]
@liquidgecka Thanks for the report.  Looks like a pretty straightforward fix if you'd like to provide a PR we would greatly appreciate that.

**Comment by liquidgecka** [comment]
It wasn't as straight forward to me. I removed the priority flag to get my stuff working but that would remove the ability for users to set priorities on rabitmq policies (we use 0 since we only have two that don't conflict). I initially suspected that something else was injecting a priority argument but when I adjusted the arg name to 'rmqpriority' the error continued to follow the name so I was out of ideas and time. =)
